ste/downloader-azureFiles_windows.go

```diff
@@ -98,5 +107,10 @@ func (bd *azureFilesDownloader) PutSDDL(sip ISMBPropertyBearingSourceInfoProvide
 		nil,
 	)
 
+	if err != nil && strings.HasPrefix(sddlString, "O:SYG:SYD:") {
+		// TODO: awaiting replies re where this SSDL comes from
+		return errorCantSetLocalSystemSddl
+	}
```

Comment:
We should get to the bottom of this before we merge this PR.

Reply:
Also, you should remove the trailing "D:" because it's possible for the DACL section to not exist (I think)

---
In reply to: [393339778](https://github.com/Azure/azure-storage-azcopy/pull/898#discussion_r393339778) [](ancestors = 393339778)

Reply:
Have removed the D.

I think I'll merge now, since it may take a few days to get to the bottom of it. We can address that in a separate PR